jlm/hls/backend/rhls2firrtl/RhlsToFirrtlConverter.cpp

```diff
@@ -1937,85 +1960,83 @@ MLIRGenImpl::MlirGen(const llvm::lambda::node * lambdaNode)
 
 // Returns a PortInfo of ClockType
 void
-MLIRGenImpl::AddClockPort(::llvm::SmallVector<circt::firrtl::PortInfo> * ports)
+RhlsToFirrtlConverter::AddClockPort(::llvm::SmallVector<circt::firrtl::PortInfo> * ports)
 {
   struct circt::firrtl::PortInfo port = {
-    builder.getStringAttr("clk"),
-    circt::firrtl::ClockType::get(builder.getContext()),
-    circt::firrtl::Direction::In,
-    {},
-    location,
+    Builder_->getStringAttr("clk"), circt::firrtl::ClockType::get(Builder_->getContext()),
```

Comment:
Each argument on a single line.

Reply:
It's clang format that places them like this